### PR TITLE
docs: hacktoberfest banner replacement

### LINKED ISSUE #121: [DOCS 📄] Compact hacktoberfest banner for README.md
### Description

- Right now the current hacktoberfest logo banner is very large and consuming a lot of space.
- Suggest a new compact logo that is similar to this original one in this README.md.

### Screenshots

_No response_

### Additional information

_No response_

### PR DESCRIPTION
**Pull request by adisuyash**
# Issue https://github.com/warmachine028/memories/issues/121 📐
- I [AdiSuyash](https://github.com/adisuyash) have worked for #121 

I have replaced the Hacktoberfest banner & did some necessary documentation changes which will provide a better user experience.

This pull request introduces two main changes:

## Changes Made
1. **Hacktoberfest Banner Replacement** : I have replaced the Old Banner of Hacktoberfest by a header LOGO.
2. **Chore** : I've re-written some words & also maintained alignment. 

## Guidelines 🔐

I accept the fact that I have followed the guidelines and have not copied the codes from around the internet

- [x] **Contribution Guidelines**
- [x] **Code of Conduct**

## Issue to be closed 🛅

- My pull request closes #121 

## Screenshots 📷
image

 [AdiSuyash](github.com/adisuyash) have worked for #121

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -33,14 +31,18 @@
...
 
--   [Introduction](#introduction)
--   [Improvements](#improvements)
--   [Tech Stack Used](#tech-stack-used)
--   [Preview](#preview)
--   [Demo](#demo)
--   [Designs](#designs)
--   [License](#license)
--   [Best Contributors](#best-contributors)
+- [Memories](#memories)
```
Comment (warmachine028) [inline README.md:34]: Restore this section

**Review by warmachine028 — CHANGES_REQUESTED** [review]
Screenshot_2023-10-16-00-10-03-940_com android chrome
Missing center alignment

**Comment by warmachine028** [comment]
See closed PRs and correct the top portion of the PR template

**Comment by warmachine028** [comment]
Make the changes first please

**Comment by adisuyash** [comment]
I have mistakenly committed these changes.
Please check again & share the feedback

**Comment by adisuyash** [comment]
> Make the changes first please

I have corrected the PR

**Comment by warmachine028** [comment]
Please center align the items

**Review by warmachine028 — APPROVED** [review]
I have made the necessary changes

**Pull request merged**